compiler/test/suites/pattern_matching.re

```diff
@@ -205,15 +221,15 @@ describe("pattern matching", ({test, testSkip}) => {
   );
   assertSnapshot(
     "constant_match_2",
-    "match ((\"foo\", 5, false)) { (\"bar\", 5, false) => false, (\"foo\", _, true) => false, (\"foo\", _, false) => true, _ => false }",
+    {|match (("foo", 5, false)) { ("bar", 5, false) => false, ("foo", _, true) => false, ("foo", _, false) => true, _ => false }|},
```

Comment:
It could be nice to split these ones onto multiple lines 🙂

Reply:
about the ones that are like that but are not using the alternate string syntax, should they be kept the same or should I break lines with the normal string syntax? as this one
![image](https://user-images.githubusercontent.com/37031640/195948388-c3a984c0-47af-455b-9fb0-f3e7c38f04b8.png)

Reply:
I think either is fine.

Reply:
updated